<ui version="4.0" >
 <class>qmwMainWindow</class>
 <widget class="QMainWindow" name="qmwMainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>File Commander</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>12</x>
      <y>13</y>
      <width>567</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QSplitter" name="qsSplitter" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QComboBox" name="qcbLeftDrive" >
             <property name="maximumSize" >
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="focusPolicy" >
              <enum>Qt::NoFocus</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="qlLeftDriveInfo" >
             <property name="text" >
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="qpbLeftRootDir" >
             <property name="maximumSize" >
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="focusPolicy" >
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text" >
              <string>\</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="qpbLeftUpDir" >
             <property name="maximumSize" >
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="focusPolicy" >
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text" >
              <string>..</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="qlLeftPath" >
             <property name="text" >
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="qswLeft" >
           <property name="currentIndex" >
            <number>0</number>
           </property>
           <widget class="QWidget" name="page" />
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="qlLeftSelected" >
           <property name="text" >
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QComboBox" name="qcbRightDrive" >
             <property name="maximumSize" >
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="focusPolicy" >
              <enum>Qt::NoFocus</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="qlRightDriveInfo" >
             <property name="text" >
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="qpbRightRootDir" >
             <property name="maximumSize" >
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="focusPolicy" >
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text" >
              <string>\</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="qpbRightUpDir" >
             <property name="maximumSize" >
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="focusPolicy" >
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text" >
              <string>..</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="qlRightPath" >
             <property name="text" >
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="qswRight" >
           <property name="currentIndex" >
            <number>0</number>
           </property>
           <widget class="QWidget" name="page_3" />
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="qlRightSelected" >
           <property name="text" >
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>TextLabel</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>4</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="qpbView" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>F3 View</string>
         </property>
         <property name="shortcut" >
          <string>F3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="qpbEdit" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>F4 Edit</string>
         </property>
         <property name="shortcut" >
          <string>F4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="qpbCopy" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>F5 Copy</string>
         </property>
         <property name="shortcut" >
          <string>F5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="qpbMove" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>F6 Move</string>
         </property>
         <property name="shortcut" >
          <string>F6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="qpbNewDirectory" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>F7 New Directory</string>
         </property>
         <property name="shortcut" >
          <string>F7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="qpbDelete" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>F8 Delete</string>
         </property>
         <property name="shortcut" >
          <string>F8</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>590</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="qaExit" />
   </widget>
   <widget class="QMenu" name="menuConfiguration" >
    <property name="title" >
     <string>&amp;Configuration</string>
    </property>
    <addaction name="qaOptions" />
    <addaction name="qaSavePosition" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuConfiguration" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
  <action name="qaExit" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="qaOptions" >
   <property name="text" >
    <string>&amp;Options</string>
   </property>
  </action>
  <action name="qaSavePosition" >
   <property name="text" >
    <string>&amp;Save position</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>qaExit</sender>
   <signal>triggered()</signal>
   <receiver>qmwMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>296</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
